test/test_tensorexpr_pybind.py

```diff
@@ -59,5 +63,34 @@ def test_external_calls(self):
             codegen.call([tA, tB, tC])
             torch.testing.assert_allclose(torch.matmul(tA, tB), tC)
 
+
+    @unittest.skipIf(not LLVM_ENABLED, "LLVM backend not enabled")
+    def test_kernel(self):
+        def f(a, b, c):
+            return a + b + c
+        device, size = 'cpu', (4, 4)
+        x = torch.rand(size, device=device)
+        y = torch.rand(size, device=device)
+        z = torch.rand(size, device=device)
+
+        graph_str = """
+graph(%a.1 : Float(4, 4, strides=[4, 1], requires_grad=0, device=cpu),
+      %b.1 : Float(4, 4, strides=[4, 1], requires_grad=0, device=cpu),
+      %c.1 : Float(4, 4, strides=[4, 1], requires_grad=0, device=cpu)):
+  %6 : int = prim::Constant[value=1]()
+  %7 : Float(4, 4, strides=[4, 1], requires_grad=0, device=cpu) = aten::add(%a.1, %b.1, %6) # test/test_tensorexpr_pybind.py:73:19
+  %3 : Float(4, 4, strides=[4, 1], requires_grad=0, device=cpu) = aten::add(%7, %c.1, %6) # test/test_tensorexpr_pybind.py:73:19
+  return (%3)
+        """
+        graph = torch._C.parse_ir(graph_str)
+
+        with kernel_arena_scope():
+            kernel = torch._C._te.TensorExprKernel(graph)
+            res1 = kernel.run((x, y, z))
```

Comment:
Do you mind adding a test with scalar arguments as well?

Reply:
Sure, I'll add one!